setup.py

```diff
@@ -361,6 +361,7 @@ def run(self):
 
     install_requires=[
         'pyaaf2>=1.4,<1.7',
+        'importlib_metadata==5.0.0; python_version < "3.8"',
```

Comment:
`importlib_metadata==5.0.0` could cause really painful to resolve conflicts to our users. What would be the lowest acceptable version? From https://pypi.org/project/importlib-metadata/#description I see hat 1.4 is equivalent to wwat's i CPython 3.8. SO I guess we could use `importlib_metadata>=1.4; python_version < "3.8"`?

Reply:
Another good call.  Updated.